src/main/java/ch/njol/skript/conditions/CondItemDespawn.java

```diff
@@ -0,0 +1,33 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Item;
+
+@Name("Will Despawn")
+@Description("Checks if the dropped item will be despawned naturally through Minecraft's timer.")
+@Examples({
+	"if all dropped items can despawn naturally:",
+		"\tprevent all dropped items from naturally despawning"
+})
+@Since("INSERT VERSION")
+public class CondItemDespawn extends PropertyCondition<Item> {
+
+	static {
+		PropertyCondition.register(CondItemDespawn.class, PropertyType.CAN, "(despawn naturally|naturally despawn)", "itementities");
+	}
```

Comment:
Can and Will? 
`if event-dropped item will despawn naturally` and `if event-dropped item can despawn naturally` seem to work